app/views/layouts/application.html.erb

```diff
@@ -11,11 +11,35 @@
   </head>
 
   <body>
+
     <% flash.each do |type, msg| %>
       <div class="flash-<%=type%>">
         <p><%=msg%></p>
       </div>
     <%end%>
+
+    <% if current_user.present? %>
+      <% flag = true %>
+      <h3>User Info:</h3>
```

Comment:
You don't need additional info here. In the future it will be something like this: 
![image](https://user-images.githubusercontent.com/35406008/199931122-847b3c23-1c92-4192-b4dc-f4bd697a69e8.png)